### PR TITLE
Upgrading Suru++ Asprómauros

### PR DESCRIPTION
**Pull request by gusbemacbe**
Hello @actionless !

I have just upgraded Suru++ Asprómauros, adding gradients support, including @Roboron3042's Cyberpunk Neon, for **all** the icons of `actions`, `devices`, `emblems`, `emotes`, `panel` and `status`, with their `symbolic` folders. 

Review the files I have just updated and see if they are all OK. If you find wrong, tell me where I have to fix.

### CODE REVIEW

**Comment by actionless** [comment]
is the script working for you after that change?

https://travis-ci.com/themix-project/oomox/jobs/205589435

File: plugins/icons_suruplus_aspromauros/change_color.sh [hunk 1]
```diff
@@ -136,21 +139,27 @@ fi
...
+		"$tmp_dir"/Suru++-Asprómauros/emotes/symbolic \
 		"$tmp_dir"/Suru++-Asprómauros/mimetypes/16 \
-		"$tmp_dir"/Suru++-Asprómauros/places/16 \
+		"$tmp_dir"/Suru++-Asprómauros/panel/{16,22,24} \
+		"$tmp_dir"/Suru++-Asprómauros/places/{16,symbolic} \
+		"$tmp_dir"/Suru++-Asprómauros/status/symbolic \
 		-type f -name '*.svg' \
 		-exec sed -i'' \
-			-e 's/currentColor/url(#oomox)/g' \
+		# For avoiding to overwrite `fill|stroke="currentColor" class="ColorScheme-Highlight"`
```
Comment (actionless) [inline plugins/icons_suruplus_aspromauros/change_color.sh]: you can't put comments between `\` escaped splitlnes

**Comment by gusbemacbe** [comment]
Hello @actionless!

It used to work before making a comment. When added a comment, I was not aware that the added comment caused Oomox to break. Sorry, I have just fixed it and it worked without the comment.

I added that comment because I wanted to explain you why those long replacements.

File: plugins/icons_suruplus_aspromauros/change_color.sh (hunk 1 above)
Comment (gusbemacbe) [reply plugins/icons_suruplus_aspromauros/change_color.sh]: Done

**Comment by actionless** [comment]
thanks a lot!

**Pull request merged**